docs/core/deploying/trimming/prepare-libraries-for-trimming.md

```diff
@@ -97,57 +95,38 @@ To create the trimming test app:
 
 If library targets a TFM that is not trimmable, for example `net472` or `netstandard2.0`, there's no benefit to creating a trimming test app. Trimming is only supported for .NET 6 and later.
 
-### [.NET 6](#tab/net6)
+:::zone pivot="dotnet-6-0"
 
 * Set `<TrimmerDefaultAction>` to `link`. <!-- only diff with .NET7+ -->
+
+:::zone-end
+
```

Comment:
The newlines here cause the rendering to come out as two separate bullet point lists:
![image](https://github.com/dotnet/docs/assets/10670590/1097c50c-41b5-481b-9969-4d32dfde808d)

Not a big deal, just weird. Not sure if the zones can actually render it as a single list... but maybe removing the newlines would help?